src/poliastro/earth/__init__.py

```diff
@@ -0,0 +1,130 @@
+from functools import partial
+
+from astropy import units as u
+
+from poliastro.bodies import Earth
+from poliastro.core.perturbations import atmospheric_drag_model
+from poliastro.twobody.orbit import Orbit
+from poliastro.twobody.propagation import cowell
+
+
+class EarthSatellite:
+
+    """
+    Position and velocity of a body with respect to Earth
+    at a given time
+    """
+
+    def __init__(self, orbit):
+        """Constructor.
+
+        Parameters
+        ----------
+        orbit : Orbit
+            Position and velocity of a body with respect to an attractor
+            at a given time (epoch).
+
+        Raises
+        ------
+        TypeError
+            If the orbit is not an Orbit's instance.
+
+        """
+
+        if not isinstance(orbit, Orbit):
```

Comment:
Nahhhh, duck typing :) Also, we might introduce gradual typing in the future to avoid these kind of errors

Reply:
However! We can check if `orbit.attractor is Earth`, that would be great